moveit_planners/chomp/chomp_interface/src/chomp_plugin.cpp

```diff
@@ -57,10 +57,16 @@ class CHOMPPlannerManager : public planning_interface::PlannerManager
     if (!ns.empty())
       nh = ros::NodeHandle(ns);
 
+    planning_interface::PlannerConfigurationMap pconfig;
     for (const std::string& group : model->getJointModelGroupNames())
     {
       planning_contexts_[group] = std::make_shared<CHOMPPlanningContext>("chomp_planning_context", group, model, nh);
+      const planning_interface::PlannerConfigurationSettings planner_config_settings{
+        group, group, std::map<std::string, std::string>()
+      };
+      pconfig[planner_config_settings.name] = planner_config_settings;
     }
+    setPlannerConfigurations(pconfig);
```

Comment:
This kind of code duplication should be avoided! Please move this code into the base class, i.e. into `PlannerManager::initialize()`!